docs/pbc/all/payment-service-providers/unzer/configuration-in-the-back-office/add-unzer-marketplace-credentials.md

```diff
@@ -13,6 +13,12 @@ Each section contains reference information. Make sure to review it before you s
 
 To add Unzer marketplace credentials, take the steps in the following sections.
 
+{% info_block infoBox "Note" %}
+
+All merchants allowed to sell their products in your stores must have Unzer credentials in the back office. Otherwise, your customers cannot buy their products; they will not be able to start the checkout process.
```

Comment:
```suggestion
All merchants allowed to sell their products in your stores must have Unzer credentials in the Back Office. Otherwise, your customers cannot buy their products; they will not be able to start the checkout process.
```